src/MSBuild/CommandLineSwitches.cs

```diff
@@ -651,6 +656,26 @@ internal string[][] GetFileLoggerParameters()
             return groupedFileLoggerParameters;
         }
 
+        /// <summary>
+        /// Checks if the provided parametrized switch needs to be unquoted.
+        /// The method will return 'true' in case:
+        ///     The changewave 17.10 is not set and
+        ///     The parametrized switch is 'Target'
+        /// </summary>
+        private bool IsMultipleAllowedSwithParameterDueToUnquote(bool unquoteParameter, ParameterizedSwitch parameterizedSwitch)
+        {
+            if (!unquoteParameter || !Traits.Instance.EscapeHatches.UnquoteTragetSwitchParameters)
+            {
+                return false;
+            }
+            if (parameterizedSwitch == ParameterizedSwitch.Target)
+            {
+                return true;
```

Comment:
Please consider document (comment) reasoning for unquoting Target and also mention msbuild issue number #xxxx

Reply:
Thank you! Added comment right before the Target parametrized switch comparison.